jina/drivers/__init__.py

```diff
@@ -258,7 +258,7 @@ def pre_traverse(_docs, traverse_on, context_doc=None):
             _traverse(docs, 'chunks')
         if 'matches' in self.traverse_fields:
             for d in docs:
-                _traverse(d.matches, 'matches')
+                _traverse(d, 'matches')
```

Comment:
no this is wrong, please revert it back.

Reply:
_traverse do not work on single doc

Reply:
yes, u r right.